### DISCUSSION TITLE
bun-v1.0.24

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#8297] [Announcements]
To install Bun v1.0.24:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.24:
```bash
bun upgrade
```

#### This release adds **[Bun Shell](https://bun.sh/blog/the-bun-shell)**

<a href="https://bun.sh/blog/the-bun-shell">

<img src="https://github.com/oven-sh/bun/assets/709451/3a6ac28c-ec19-463c-8d7e-13383549219a" height=400 />

</a>

<br />

## Thanks to 14 contributors!

- [@Aarav-Juneja](https://github.com/Aarav-Juneja)
- [@cena-ko](https://github.com/cena-ko)
- [@DontBreakAlex](https://github.com/DontBreakAlex)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Electroid](https://github.com/Electroid)
- [@guest271314](https://github.com/guest271314)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@keepdying](https://github.com/keepdying)
- [@LukasKastern](https://github.com/LukasKastern)
- [@maxmilton](https://github.com/maxmilton)
- [@nektro](https://github.com/nektro)
- [@paperdave](https://github.com/paperdave)
- [@RiskyMH](https://github.com/RiskyMH)
- [@yus-ham](https://github.com/yus-ham)
- [@zackradisic](https://github.com/zackradisic)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.24'>bun-v1.0.24</a>.</em>

**Comment by leeoniya** [comment]
@Jarred-Sumner the bun shell blog post doesnt show the startup perf. only that bash and sh are 7ms

**Comment by AhmedBaset** [comment]
How about installing on Windows? (without wsl)

**Comment by truongan07** [comment]
Very cool, i can use it to convert type of image to webp, optimize image, interact redis & more. Because i used bun ffi to do this and hard. Thanks Sumner.
